model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -1086,6 +1086,16 @@ case class WorkspaceStatus(workspaceName: WorkspaceName, statuses: Map[String, S
 
 case class BucketUsageResponse(usageInBytes: BigInt, lastUpdated: Option[DateTime])
 
+case class BucketMetricsResponse(
+  metrics: Seq[BucketMetric]
+)
+
+//See https://cloud.google.com/storage/docs/storage-classes
+//I believe the options for storageclass are STANDARD, NEARLINE, COLDLINE, ARCHIVE, REGIONAL, MULTI-REGIONAL and DRA.
+//I've seen REGIONAL and MULTI_REGIONAL in practice; I have not seen DRA so I don't know exactly how it would be coded
+//Hence I'm afraid to make an enumeration until I can verify what's possible to see.
+case class BucketMetric(storageClass: String, valueInBytes: Double)
```

Comment:
I believe all we care about is `storageClass` and `valueInBytes`; orch can then calculate cost based on storage class.  There might be duplicates of storage class, as soft-deleted objects should be separated out, and REGIONAL and MULTI-REGIONAL both count as STANDARD, but I don't think we should see any actual double-counting as long as the interval we look over isn't more than 5 minutes.